src/libprojectM/projectM.cpp

```diff
@@ -929,6 +929,7 @@ void projectM::selectNext(const bool hardCut) {
         }
         else {
             // otherwise move forward 
+            renderer->m_activePresetID++;
```

Comment:
can it exceed the bounds?

Reply:
No that's covered already. A few lines above there is a check to see if we hit the boundary. If so, it will rotate back to the top when you reach the bottom.